app/assets/javascripts/govuk_publishing_components/analytics-ga4/pii-remover.js

```diff
@@ -27,6 +27,10 @@
   var UNLOCK_TOKEN_PATTERN = /unlock_token=[a-zA-Z0-9-]+/g
   var STATE_PATTERN = /state=.[^&]+/g
 
+  // some users mistakenly use GOV.UK to search for the status of their applications.
```

Comment:
Suggestion: might help to add to this comment an example of the numbers this should match.

Reply:
Although I've just noticed there's some in the spec file, so 🤷

Reply:
No worries, will add more specific examples then merge 👍